service/follower/consensus_internal_test.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2021 Optakt Labs OÜ
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package follower
+
+import (
+	"io"
+	"testing"
+
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/optakt/flow-dps/testing/helpers"
+)
+
+func TestNewConsensus(t *testing.T) {
+	log := zerolog.New(io.Discard)
+	db := helpers.InMemoryDB(t)
+
+	// FIXME
```

Comment:
Perhaps something more elaborate would be nice :D

Reply:
I feel like it's best to just remove all tests again, because there is practically nothing of relevance left that can be reused.